### ISSUE TITLE
Query params without quotes don't work unless "quoted string" option is enabled

### ISSUE THREAD

**Issue by klanchman** [TablePlus/TablePlus#2829]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
PostgreSQL 11.15

3. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
Version 5.0.0 (454)

5. The steps to reproduce this issue:

- Make a simple query with a query param without quotes around it, e.g. `select * from users where id = $1;`
- Enable query params & set the regex as needed:
<img width="300" alt="Screen Shot 2022-10-04 at 10 52 48 AM" src="https://user-images.githubusercontent.com/4845407/193852487-e5b2edd1-2759-449a-b079-2c3f2d7e932c.png">
<img width="300" alt="Screen Shot 2022-10-04 at 10 53 07 AM" src="https://user-images.githubusercontent.com/4845407/193852537-bb5c19e6-9ed1-4bbc-84b6-1c01361d7b9c.png">

- Run the query

Expected: TablePlus prompts for query param `$1`
Actual: 
```
Query 1 ERROR: ERROR:  there is no parameter $1
LINE 1: select * from users where id = $1;
                                       ^
```

If I enable "Allow searching for variables in Quoted String", then it works as expected. But there's no quoted string in the query 😄

**Comment by huyphams** [comment]
ops that were bad. Let me fix it.

**Comment by huyphams** [comment]
Hi @klanchman if you have a chance, please try this patch: https://www.dropbox.com/s/3rkgc4yylrr9buj/TablePlus.dmg?dl=0
Please let me know if you're still seeing the issue.

**Comment by klanchman** [comment]
Hi @huyphams, thanks for the quick response!

I gave that build a try, it looks like it fixes my initial issue, but now I think there's a new issue?

I have "Allow searching for variables in Quoted String" disabled, and I ran this query: `select * from users where id = '$something';`

I think in older versions this would not count as a parameter, but now regardless of my Quoted String setting TablePlus is prompting for a parameter. I'm not sure if that's expected or not?
